docker/zookeeper/zookeeper-conf/zoo.cfg

```diff
@@ -3,4 +3,4 @@ initLimit=10
 syncLimit=5
 dataDir=/data/zookeeper
 clientPort=2181
-maxClientCnxns=200
+maxClientCnxns=800
```

Comment:
we need to double check that before release on prod

Reply:
noted